### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by Neustradamus** [thii/FontAwesome.swift#288]
Dear @thii, @LukasHromadnik, @oliverdowling, @shams-ahmed, @OlafAndreas, @MatrixSenpai, @doedje, @nanoxd, @novinfard, @marcelofabri, @sandeepbol, @DwayneCoussement, @mpclarkson, @antranapp, @ykphuah, @lukasackee, @rbresjer, @antonyalkmim, @Tulleb, @Imperiopolis, @MoElnaggar14, @koenpunt, @yoiang, @stNamco, @bobbyleary, @himaratsu, @CAPIStkidd, @maxep, @szehnder, @Rich86man, @Maik639, @Hendrik44, @hslightnin, @kevinxh, @arnaudlamy, @muyexi, @timothybarraclough, @alexkafer, @mimu0, @ecnepsnai, @xslim, @ReadmeCritic, @morizotter, @King-Wizard, @joubin, @katunch, @prrrnd, @PiB02, @BenchR267,

It is possible to have new releases?

The latest FontAwesome.swift is 1.9.0 (2020-03-28).

At this time, the current latest FontAwesome is 6.4.2 (2023-08-02):
- https://github.com/FortAwesome/Font-Awesome/releases

Maybe it is time to move in organization to wake up?

Thanks in advance.

**Comment by doedje** [comment]
We'll happily await your PR...

**Comment by Tulleb** [comment]
> Maybe it is time to move in organization to wake up?

Maybe it's time for your to stop taking open source repos for granted? 🤡 

FYI you can switch to SF Symbols image gallery very easily now.

**Comment by Neustradamus** [comment]
It is linked to this comment:
- https://github.com/freeotp/freeotp-ios/issues/309#issuecomment-1745672148

**Comment by Neustradamus** [comment]
I am not alone ^^
- https://github.com/thii/FontAwesome.swift/issues/287

**Comment by OlafAndreas** [comment]
This is an open source project originally built by thii, and expanded on by various contributors. There is no single person responsible to make this project work for anyone. If you see something that needs attention, you are expected to take initiative and make that change and possibly merge the changes and create a new release to make it available for others.

@Neustradamus please understand that previous contributors offered their help because they needed a change/addition to be made. It seems to me like it is your time to make that change.

And by the way, I will link you to the license for this project, you can find it [here](https://github.com/thii/FontAwesome.swift/blob/master/LICENSE).

**Comment by Neustradamus** [comment]
@OlafAndreas: Yes I know, I have contacted contributors (like you) to request help to have new changes, new builds... :)

**Comment by OlafAndreas** [comment]
I will happily review a pull request if you make the changes you want included in the next release.
